.rubocop.yml

```diff
@@ -1,515 +1,6 @@
-    branches.
-  Enabled: false
-Performance/Count:
-  Description: Use `count` instead of `select...size`, `reject...size`, `select...count`,
-    `reject...count`, `select...length`, and `reject...length`.
-  Enabled: false
-Performance/Detect:
-  Description: Use `detect` instead of `select.first`, `find_all.first`, `select.last`,
-    and `find_all.last`.
-  Reference: https://github.com/JuanitoFatas/fast-ruby#enumerabledetect-vs-enumerableselectfirst-code
-  Enabled: false
-Performance/FlatMap:
-  Description: Use `Enumerable#flat_map` instead of `Enumerable#map...Array#flatten(1)`
-    or `Enumberable#collect..Array#flatten(1)`
-  Reference: https://github.com/JuanitoFatas/fast-ruby#enumerablemaparrayflatten-vs-enumerableflat_map-code
-  Enabled: false
-Performance/ReverseEach:
-  Description: Use `reverse_each` instead of `reverse.each`.
-  Reference: https://github.com/JuanitoFatas/fast-ruby#enumerablereverseeach-vs-enumerablereverse_each-code
-  Enabled: false
-Performance/Size:
-  Description: Use `size` instead of `count` for counting the number of elements in
-    `Array` and `Hash`.
-  Reference: https://github.com/JuanitoFatas/fast-ruby#arraycount-vs-arraysize-code
-  Enabled: false
-Performance/StringReplacement:
-  Description: Use `tr` instead of `gsub` when you are replacing the same number of
-    characters. Use `delete` instead of `gsub` when you are deleting characters.
-  Reference: https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code
-  Enabled: false
-Rails/ActionFilter:
-  Description: Enforces consistent use of action filter methods.
-  Enabled: false
-Rails/Date:
-  Description: Checks the correct usage of date aware methods, such as Date.today,
-    Date.current etc.
-  Enabled: false
-Rails/FindBy:
-  Description: Prefer find_by over where.first.
-  Enabled: false
-Rails/FindEach:
-  Description: Prefer all.find_each over all.find.
-  Enabled: false
-Rails/HasAndBelongsToMany:
-  Description: Prefer has_many :through to has_and_belongs_to_many.
-  Enabled: false
-Rails/Output:
-  Description: Checks for calls to puts, print, etc.
-  Enabled: false
-Rails/ReadWriteAttribute:
-  Description: Checks for read_attribute(:attr) and write_attribute(:attr, val).
-  Enabled: false
-Rails/ScopeArgs:
-  Description: Checks the arguments of ActiveRecord scopes.
-  Enabled: false
-Rails/TimeZone:
-  Description: Checks the correct usage of time zone aware methods.
-  StyleGuide: https://github.com/bbatsov/rails-style-guide#time
-  Reference: http://danilenko.org/2012/7/6/rails_timezones
-  Enabled: false
-Rails/Validation:
-  Description: Use validates :attribute, hash of validations.
-  Enabled: false
-Rails:
-  Enabled: true
-Metrics/PerceivedComplexity:
-  Description: A complexity metric geared towards measuring complexity for a human
-    reader.
-  Enabled: true
-  Max: 7
+inherit_from:
+  - https://raw.githubusercontent.com/riboseinc/oss-guides/master/ci/rubocop.yml
```

Comment:
I think Hound CI used to have some issues with remote rubocop file, could you double check that it actually works with our style guidelines?

Reply:
I have checked, during several PRs and it looks working.

One strange note that Hound for some reason fine more issues, for example, it found "Redundant curly braces around a hash parameter." while local `rubycop` wasn't

Reply:
If that's the case - then I would suspect hound is probably using the default rubocop config or the one in hound app instead of our one 🤔

Reply:
@abunashir definitely not, because in the default config: line length is 80, in out 120 and hound uses correct one (120)

Reply:
Cool, in that case all good 👍🏼